Add `CONTRIBUTING.md` (#1935)
radicle-dev/radicle-upstream: README.md

A good entry point for exploration is [`DEVELOPMENT.md`][de], where you'll find
information on how to build Upstream from source.

> If you're looking to contribute, take a look at [`CONTRIBUTE.md`][co] to learn

Reviewer: ```suggestion
If you would like to contribute, take a look at [`CONTRIBUTE.md`][co] to learn
```

> about the different ways that we accept contributions.

Reviewer: ```suggestion
about the different ways in which we accept contributions.
```
